c++/src/kj/compat/http.c++

```diff
@@ -4084,6 +4099,11 @@ private:
       KJ_FAIL_ASSERT("Bytes are not counted for the individual states of WebSocketPipeImpl.");
     }
 
+    kj::Maybe<kj::String> getPreferredExtensions(ExtensionsContext ctx) override {
+      KJ_FAIL_ASSERT(
+          "BlockedReceive WebSocket has no websocket to get a compression preference from.");
```

Comment:
I wonder if in this case we should forward the call to the `pipe`, but I was under the impression we would have called _from_ the pipe into the `BlockedReceive` websocket, so calling back into the `pipe` would be circular.